internal/util/auth/auth.go

```diff
@@ -61,3 +79,33 @@ func LoadConfigAndAuth(ctx context.Context, configManager *config.ClientConfigMa
 	}
 	return usecases.NewAuthUsecase(configManager, force, silent).Run(ctx)
 }
+
+func acquireLock(_ context.Context) (mutex.Releaser, error) {
+	s := spinner.NewSpinner()
```

Comment:
If the auth is silent there might shouldn't be a spinner?

Reply:
good catch will if'it